src/main/java/com/ua/test_task/service/operation/OperationInitializationService.java

```diff
@@ -75,4 +76,16 @@ public static OrderOperationType parseOrderOperationType(String type) {
         }
         return null;
     }
+
+    public static BidOrAskOperation parseUpdateOperation(String type) {
```

Comment:
Change the method title to `parseUpdateOperationType`

Reply:
fix